README.rst

```diff
@@ -143,7 +143,27 @@ Available formatting constants are::
 
     Fore: BLACK, RED, GREEN, YELLOW, BLUE, MAGENTA, CYAN, WHITE, RESET.
     Back: BLACK, RED, GREEN, YELLOW, BLUE, MAGENTA, CYAN, WHITE, RESET.
-    Style: DIM, NORMAL, BRIGHT, RESET_ALL
+    Style: DIM, NORMAL, BRIGHT, BRIGHT_OFF, REVERSE, UNDERLINE, RESET_ALL
+
+``Style.REVERSE_OFF`` and ``Style.UNDERLINE_OFF`` are provided to allow intent
+to be clearer.
+They currently have the same values as ``REVERSE`` and ``UNDERLINE``,
+respectively. Using ``REVERSE`` a 2nd time will toggle REVERSE off again
+(and the same for UNDERLINE).
+
+It's important to realise that ``Style.UNDERLINE_OFF`` and ``Style.REVERSE_OFF``
+are not guaranteed to turn the relevant effect off.  If you've called
+``colorama.init`` with ``autoreset=False``, and left the terminal in underline
+mode, then the next time either ``Style.UNDERLINE`` or ``Style.UNDERLINE_OFF``
+is encountered, underlining will be turned off.
+
+Note that REVERSE and UNDERLINE require Windows 10, they don't have any effect
+on Windows 7 or 8.
```

Comment:
Technically it's not the Windows version but the specific terminal (cmd) on these Windows versions.
(did we ever try these values on PowerShell on older versions? It's not important, anyway).

Reply:
True. This text will be removed when I fix the problem with turning the attributes off in Windows.